docs/portal_config.md

```diff
@@ -91,6 +91,14 @@ Below is an example, with inline comments describing what each JSON block config
         }
       ]
     },
+    "emailSignUpForm": { // optional; used to show a govdelivery.com Email SignUp Form on the homepage
```

Comment:
is this feature also applicable to other envs?

if this is an NCT specific thing, we should make these hardcoded, instead of keep enlarging the portal config file

Reply:
currently NCT specific, currently the action and subscribeToTopicId are the only required values everything else has defaults set. Do you think this would be better entirely hard coded on the nctPage? or do you just mean to have an option to disable enable it here?

Reply:
I would just hardcode the entire thing, or at most have one option to turn it on/off just in case the sponsors change their minds 🤷‍♂️ 

But if you want to have an option then I suggest to name it as `nctSignupForm` so others know it is for NCT only

Reply:
Sorry id not understand you at first. It could be used for any govdelivery email signup, dont know if more services could use it in the future that is why i made it extremely flexible. Next time im asked to do something like this ill do it the fast way as i have already spent too much time on this

Reply:
No other env needs the govdelivery signup now and if they will need something to subscribe, it is not necessarily from govdelivery

That is why I suggest to make it less customizable, and place it only in `nctPage` for now